utils/nativefuncs.go

```diff
@@ -242,6 +243,33 @@ func RegisterNativeFuncs(vm *jsonnet.VM, resolver Resolver) {
 			return ret, nil
 		},
 	})
+
+	vm.NativeFunction(&jsonnet.NativeFunction{
+		Name:   "validateJSONSchema",
+		Params: []jsonnetAst.Identifier{"obj", "schema"},
+		Func: func(args []interface{}) (interface{}, error) {
+			obj := args[0]
+			schema := args[1]
+
+			jSchema, err := json.Marshal(schema)
+			if err != nil {
+				return nil, fmt.Errorf("unable to json marshal schema: %w\n", err)
+			}
+
+			// No URL defaults to draft 7 of JSONSchema
+			sch, err := jsonschema.CompileString("", string(jSchema))
+			if err != nil {
+				return false, fmt.Errorf("unable to compile jsonschema: %w\n", err)
```

Comment:
anyway, let's remove newline in the error message (any linter would have told you that, unfortunately we're not running a linter in the CI job)

```suggestion
				return false, fmt.Errorf("unable to compile jsonschema: %w", err)
```